tox.ini

```
@@ -99,10 +99,11 @@ deps =
   coverage
   django22: Django>=2.2,<3.0
   django30: Django>=3.0,<3.1
+  django31: Django==3.1rc1
```

TODO: I don't know how to get tox to install a pre-release version, so have pinned.

---

Django 3.1 has now been released, so this can be changed to:

```suggestion
  django31: Django>=3.1,<3.2
```